admin/database/postgres/postgres.go

```diff
@@ -352,24 +358,42 @@ func (c *connection) FindProjectByName(ctx context.Context, orgName, name string
 	if err != nil {
 		return nil, parseErr("project", err)
 	}
-	return res.AsModel()
+	return res.AsModel(c.encKeyring)
 }
 
 func (c *connection) InsertProject(ctx context.Context, opts *database.InsertProjectOptions) (*database.Project, error) {
 	if err := database.Validate(opts); err != nil {
 		return nil, err
 	}
 
+	// encrypt the prod variables and store the key id used for encryption
+	if len(c.encKeyring) == 0 {
+		return nil, errors.New("no encryption key found")
+	}
```

Comment:
this was just a defensive check, actually admin starts to fail if keyring does not have any keys. I did this so that we don't miss encryption if there is some misconfiguration or we forgot to include encryption keys. What behaviour do we want here ?

Reply:
I think we should allow the empty keyring for now

Reply:
ok